src/dosext/mfs/mfs.c

```diff
@@ -4907,6 +4948,50 @@ static int dos_fs_redirect(struct vm86_regs *state)
       Debug0((dbg_fd, "Passing %d to PRINTER SETUP CALL\n", (int)WORD(state->ebx)));
       return REDIRECT;
 
+    case LONG_SEEK: {
+      uint64_t* p_seek_struct;
+      uint64_t seek;
+      f = &open_files[sft_fd(sft)];
```

Comment:
We usually do:
```
      cnt = sft_fd(sft);
      if (cnt >= MAX_OPENED_FILES)
          return FALSE;
      f = &open_files[cnt];
```
to avoid the possibility of array overrun.